src/ng/directives/resizable.js

```diff
@@ -44,26 +45,28 @@ class Resizable extends Directive(RESIZABLE_NAME, {stickyCore: `^^?${STICKY_CORE
 
 		const sticky = this.stickyCore.sticky;
 		const context = this.context;
-
-		if (sticky && !context.sourceColumn) {
-			context.sourceColumn = sticky
+		if (sticky) {
+			context.source = sticky
 				.th(sticky.source)
 				.find(th => th.classList.contains(this.th.column.key));
 		}
 
-		const width = parseInt(this.$element[0].clientWidth);
-		context.startX = e.screenX - width;
-
+		const style = window.getComputedStyle(this.$element[0], null);
+		//context.min = parseFloat(style.getPropertyValue('min-width'));
+		context.width = parseFloat(style.getPropertyValue('width'));
```

Comment:
why do you return x and width to context? it's calculating just in moment of mousedown. and then const for drag